.github/workflows/test.yaml

```diff
@@ -120,6 +120,17 @@ jobs:
         uses: jbergstroem/hadolint-gh-action@v1
         with:
           dockerfile: "${{ env.Dockerfiles }}"
+  lint-metadata:
+    name: Lint metadata.yaml
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install check-jsonschema
+        run: python3 -m pip install check-jsonschema
+      - name: Download schema
+        run: curl -o $PWD/metadata.schema https://raw.githubusercontent.com/canonical/operator-workflows/main/.github/files/metadata.schema
```

Comment:
Add `-sfSL` to curl? `-s`: silence the download progress; `-S`: print error in silence mode; `-f` fail on server errors; `-L` follow redirects.

Reply:
Done, thx